+euphonic/ForceConstants.m

```diff
@@ -5,46 +5,60 @@
     properties(Access=protected)
         pyobj = [];  % Reference to python object
         eu = [];     % Reference to euphonic module
-        classname = 'ForceConstants';
+        classname = 'euphonic.ForceConstants'; 
```

Comment:
`ForceConstants` class docstring needs to be updated to match CoherentCrystal
```
To obtain help on this class and its methods please type:
    %
    %   >> import euphonic.doc
    %   >> doc euphonic.ForceConstants...
```
etc.